src/averageAges.js

```diff
@@ -36,8 +53,15 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = getSex(people, 'f');
+
+  const womenWithChild = withChildren
+    ? women.filter(person => getAllMother(people, person))
+    : women;
```

Comment:
Let's improve performance here and use only filter + some, instead of 2 filters + find

Reply:
I don't understand, I use 1 filter